sys/include/net/ng_pktbuf.h

```diff
@@ -139,11 +140,24 @@ void ng_pktbuf_hold(ng_pktsnip_t *pkt, unsigned int num);
 
 /**
  * @brief   Decreases ng_pktsnip_t::users of @p pkt atomically and removes it if it
- *          reaches 0.
+ *          reaches 0 and reports a possible error through an error code, if
+ *          @ref net_ng_neterr is included.
  *
  * @param[in] pkt   A packet.
+ * @param[in] err   An error code.
  */
-void ng_pktbuf_release(ng_pktsnip_t *pkt);
+void ng_pktbuf_release_error(ng_pktsnip_t *pkt, uint32_t err);
```

Comment:
the release error is always present, even if the `neterr` module is not activated?